packages/models/proto/v1/authorization/client.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto2";
+
+package authorization;
+
+message ClientV1 {
+  required string id = 1;
+  required string consumerId = 3;
+  required string name = 4;
+  optional string description = 5;
+  repeated string relationships = 6;
+  repeated ClientPurposesEntryV1 purposes = 7;
+  required ClientKindV1 kind = 8;
+  optional int64 createdAt = 9; // to do double-check why optional
+  repeated string users = 10; 
+}
+
+message ClientPurposesEntryV1 {
+  required ClientStatesChainV1 states = 2;
```

Comment:
```suggestion
  required ClientStatesChainV1 states = 1;
```

Reply:
Since it's already defined like this I think we should keep it in order to maintain the same structure